src/components/popover/Popover.js

```diff
@@ -169,7 +177,7 @@ Popover.defaultProps = {
   active: true,
   backdrop: 'dark',
   direction: 'south',
-  fullHeight: false,
+  fullHeight: true,
```

Comment:
`fullHeight` is `true by default, as this has always been its standard behaviour.